src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -0,0 +1,385 @@
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Supplier',
+      accessor: 'origin',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Destination',
+      accessor: 'destination',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Ordered On',
+      accessor: 'dateOrdered',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+      Cell: row => (
+        <span>{dateParser(row.original.dateOrdered)}</span>
+      ),
+      Footer: (
+        <span>
+          <Translate id="react.purchaseOrder.totalAmount.label" defaultMessage="Total amount" />: {totalPrice} USD
+        </span>
+      ),
+      footerClassName: 'border-0',
+    },
+  ];
+
+
+  return (
+    <div className="purchase-order-list-list-section">
+      <div className="title-box d-flex justify-content-between align-items-center">
+        <span>
+          <Translate id="react.purchaseOrder.listOrders.label" defaultMessage="List Orders" />
+          &nbsp;
+          (<Translate id="react.purchaseOrder.totalAmount.label" defaultMessage="Total amount" />: {totalPrice} USD)
+        </span>
+        <div className="btn-group">
+          <Button
+            defaultLabel="Export"
+            label="react.purchaseOrder.export.button.label"
+            variant="dropdown"
+            EndIcon={<RiDownload2Line />}
+          />
+          <div className="dropdown-menu dropdown-menu-right nav-item padding-8" aria-labelledby="dropdownMenuButton">
+            <a href="#" className="dropdown-item" onClick={() => downloadOrders(true)} role="button" tabIndex={0}>
+              <Translate
+                id="react.purchaseOrder.export.orderLineDetails.label"
+                defaultMessage="Export order line details"
+              />
+            </a>
+            <a className="dropdown-item" onClick={() => downloadOrders(false)} href="#">
+              <Translate
+                id="react.purchaseOrder.export.orders.label"
+                defaultMessage="Export orders"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <ReactTable
+        data={ordersData}
+        ref={tableRef}
+        columns={columns}
+        loading={loading}
+        pages={pages}
+        defaultPageSize={10}
+        manual
+        className="-striped -highlight zoneTable"
+        resizable={false}
+        sortable
+        collapseOnSortingChange={false}
+        previousText={<i className="fa fa-chevron-left" aria-hidden="true" />}
+        nextText={<i className="fa fa-chevron-right" aria-hidden="true" />}
+        pageText=""
+        onFetchData={(state) => {
+          const offset = state.page > 0 ? (state.page) * state.pageSize : 0;
+          const sortingParams = state.sorted.length > 0 ?
+            {
+              sort: state.sorted[0].id,
```

Comment:
I think, that by default it should sort by `orderedOn` descending (or by OrderSummaryStatus if there would be a sortOrder defined there)